### PR TITLE
ADR-795 Tests added for the guidance page

### PR DESCRIPTION
**Pull request by Kavitakum**
Added Acceptance Tests for Guidance page

### CODE REVIEW

File: src/test/resources/features/dutySuspendedDeliveries/DutySuspendedDeliveries.feature [hunk 1]
```diff
@@ -14,7 +14,9 @@ Feature: Alcohol Duty Return - Duty Suspended Deliveries
     Then I am presented with the "Declare Duty Suspended Deliveries Page"
     When I select radio button "Yes" on "Declare Duty Suspended Deliveries Page"
     And I click save and continue button on "Declare Duty Suspended Deliveries Page"
-#   Then I am presented with the "DSD Guidance Page"
+    Then I am presented with the "Duty Suspended Deliveries Guidance Page"
```
Comment (yasaracar) [inline src/test/resources/features/dutySuspendedDeliveries/DutySuspendedDeliveries.feature:17]: Do we need this Scenario at all? Doesn't happy path scenario below cover this?

File: src/test/scala/uk/gov/hmrc/alcoholDuty/pages/BasePage.scala [hunk 2]
```diff
@@ -92,6 +92,8 @@ trait BasePage extends Page with Matchers with BrowserDriver with Eventually wit
 
   def clickSaveAndContinueButton(): Unit = click on id("saveAndContinueButton")
 
+  def clickPageContinueButton(): Unit = click on id("continueButton")
```
Comment (yasaracar) [inline src/test/scala/uk/gov/hmrc/alcoholDuty/pages/BasePage.scala:95]: There is a clickContinueButton() above. Having them both at the same time might create confusion. Can we remove the other one if it is not used? If used, rename it to Submit maybe as it looks for submit id?

**Review by yasaracar — CHANGES_REQUESTED** [review]

File: src/test/resources/features/dutySuspendedDeliveries/DutySuspendedDeliveries.feature (hunk 1 above)
Comment (Kavitakum) [reply src/test/resources/features/dutySuspendedDeliveries/DutySuspendedDeliveries.feature:17]: As discussed, removed the happy path scenario and added the steps in this scenario

File: src/test/scala/uk/gov/hmrc/alcoholDuty/pages/BasePage.scala (hunk 2 above)
Comment (Kavitakum) [reply src/test/scala/uk/gov/hmrc/alcoholDuty/pages/BasePage.scala]: Renamed the clickContinueButton()  to clickSubmitButton() and changed the steps as well accordingly

**Review by yasaracar — APPROVED** [review]

**Pull request merged**